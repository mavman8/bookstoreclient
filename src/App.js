import axios from "axios";
import React from "react";
import ReactDOM from "react-dom";
import baseUrl from './config'

console.log(baseUrl);

const App = () =>{
    //127.0.0.1:8080/
    axios(`${baseUrl}/api/v1/books`).then(books=>{
        console.log(books);
    })
    return <div>My first component</div>
}

export default App;