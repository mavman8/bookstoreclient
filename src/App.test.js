import React from 'react';
import {render } from "@testing-library/react";
import App from './app';

describe('App Component', ()=>{ 

    it('should render without error', ()=>{

        const { asFragment } = render(<App/>);
        expect(asFragment()).toMatchSnapshot(); 

    });

});
