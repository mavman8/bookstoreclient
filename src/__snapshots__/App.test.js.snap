// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component should render without error 1`] = `
<DocumentFragment>
  <div>
    My first component
  </div>
</DocumentFragment>
`;
